app/src/main/java/agency/five/codebase/android/movieapp/ui/component/MovieCategoryLabel.kt

```diff
@@ -0,0 +1,78 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+
+import agency.five.codebase.android.movieapp.ui.theme.Gray700
+import androidx.annotation.StringRes
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+
+sealed class MovieCategoryLabelTextViewState()
+data class MovieCategoryStringParam(val value: String) : MovieCategoryLabelTextViewState()
+data class MovieCategoryStringResource(@StringRes val value: Int) :
+    MovieCategoryLabelTextViewState()
+
+data class MovieCategoryLabelViewState(
+    val itemId: Int,
+    val isSelected: Boolean,
+    val categoryText: MovieCategoryLabelTextViewState
+)
+
+@Composable
+fun MovieCategoryLabel(
+    movieCategoryLabelViewState: MovieCategoryLabelViewState,
+    modifier: Modifier = Modifier,
+    isSelected: Boolean,
```

Comment:
okay, you have isSelected boolean within MovieCategoryLabelViewState, so you won't need this one :)